recipes/cassandra-cpp-driver/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+    "2.15.3":
+        folder: all
```

nit: 2 characters indent
```suggestion
  "2.15.3":
    folder: all
```